### PR TITLE
Added Unifi direct doc.

### PR DESCRIPTION
**Pull request by w1ll1am23**
**Description:**
Added documentation on the Unifi direct device tracker. 

I believe the rainbird file was modified because of line ending? I am submitting this from a Unix system and maybe it was pushed via Windows? I did a clean branch from home-assistant.github.io's `next` branch and this file shows as modified without me doing anything at all. 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#10097

## Checklist:

  - [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
If you didn't touch it and include it in the commit, it can't have a diff, if your base is the same as upstream next. So either of them is not true.

File: source/_components/device_tracker.unifi_direct.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: unifi_direct
+    host: YOUR_AP_IP_ADDRESS
+    username: YOUR_USERNAME
+    password: YOUR_PASSWORD
+```
+
+Configuration variables:
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.unifi_direct.markdown]: Preferably use the new configuration tag:
https://home-assistant.io/developers/documentation/standards/#component-and-platform-pages

File: source/_components/device_tracker.unifi_direct.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+title: "Ubiquiti Unifi direct AP"
+description: "Instructions how to use a Unifi WAP as a device tracker."
+date: 2017-11-17 14:59
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ubiquiti.png
+ha_category: Presence Detection
+ha_release: 0.58
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.unifi_direct.markdown]: 0.59

**Comment by w1ll1am23** [comment]
@MartinHjelmare I tested again after opening this. I did a git clone of this repo into a clean directory. As soon as it finished I CDd into the directory. I did a git status and it says the file has changed. Everything matches up to what's going on here https://stackoverflow.com/questions/10017885/why-does-git-insist-that-a-file-has-been-modified-even-after-a-git-checkout

**Comment by MartinHjelmare** [comment]
Ok, I can reproduce the the change on the rainbird file by pulling upstream next. I'll try to fix that in another PR.

**Comment by MartinHjelmare** [comment]
Line endings fix here:
https://github.com/home-assistant/home-assistant.github.io/pull/4009

**Comment by MartinHjelmare** [comment]
@w1ll1am23 you were right about the windows line endings being the cause. But even though that file was changed you don't need to include it in your commit. So it should be possible to make this PR without that change.

**Comment by w1ll1am23** [comment]
@MartinHjelmare Thanks, that worked.

**Pull request merged**